m4_sessions/1711-inning/Group_8.md

```diff
@@ -1,9 +1,31 @@
-Group Eight
+Group Eight 
 
-Session Title: Description: Why should students come to this session?
+Session Title: Transforming your Narrative through Growth Mindset
 
-Session Outcomes: What will students leave with or gain from this session?
+Description: Why should students come to this session?
+
+Session Outcomes: What will students leave with or gain from this session? 
+
+  Students will be able to:
+    Identify when they are in a fixed mindset.
+    Identify techniques to remove themselves from the fixed mindset into a growth mindset. 
+    Identify an area that they want to transition into a growth mindset.
+    Express the difference between having a growth mindset and remaining constantly positive.
+    Express their failures through the lens of a growth mindset, and understand their failure as a learning tool. 
+    Support other students who are currently in a fixed-mindset. 
+  
 
 Session Outline: Include the flow of the session and the time for each section.
+  - Introductions/Why we decided to provide this Gearup.
+  - Provide resources on how to identify a fixed mindset. 
+  - Students will identify an area that they are in a fixed mindset.
+  - Discuss the power of your self narrative. 
```

Comment:
Will you each be sharing your self narrative?  how will you build this so that students are willing to jump in?  How vulnerable do you want them to be?